tests/e2e/fixtures/fixture-loaders.js

```diff
@@ -334,7 +352,76 @@ const deleteBlockPages = ( ids ) => {
 	);
 };
 
+/**
+ * Ensure attributes are removed from the site before setting up. This exists in case
+ * the teardown didn't happen properly for any reason. This function exists mostly as a
+ * dev-ex improvement tool while working on tests.
+ *
+ * @return {Promise<void>}
+ */
```

Comment:
Can we call teardown manually to accommodate for this need? Are we just clearing attributes and not everything else? Alternatively, can we add attributes in a more defensive way so an error won't be returned if attributes already exist? If that's not possible, can we call all teardown functions in the setup but guard that in a flag that's only true when working locally?

The issue I have with this is that this adds extra load to tests, and doesn't feel like it's enough, which might result in us adding more teardown calls in setup and hindering our E2E speed.

Reply:
I've removed the `ensureCleanAttributes` function and created a new issue for handling the clearing of the attributes https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/5410